modules/UI/util/MessageHandler.js

```diff
@@ -448,31 +452,22 @@ var messageHandler = {
      * @param messageKey the key from the language file for the text of the
      * message.
      * @param messageArguments object with the arguments for the message.
-     * @param options passed to toastr (e.g. timeOut)
+     * @param optional configurations for the notification (e.g. timeout)
      */
     participantNotification: function(displayName, displayNameKey, cls,
                     messageKey, messageArguments, options) {
-
-        // If we're in ringing state we skip all toaster notifications.
-        if(!notificationsEnabled || APP.UI.isOverlayVisible())
+        // If we're in ringing state we skip all notifications.
+        if (!notificationsEnabled || APP.UI.isOverlayVisible()) {
             return;
-
-        var displayNameSpan = '<span class="title" ';
-        if (displayName) {
-            displayNameSpan += ">" + UIUtil.escapeHtml(displayName);
-        } else {
-            displayNameSpan += "data-i18n='" + displayNameKey + "'>";
         }
-        displayNameSpan += "</span>";
-        let element = toastr.info(
-            displayNameSpan + '<br>' +
-            '<span class=' + cls + ' data-i18n="' + messageKey + '"' +
-                (messageArguments?
-                    " data-i18n-options='"
-                        + JSON.stringify(messageArguments) + "'"
-                    : "") + "></span>", null, options);
-        APP.translation.translateElement(element);
-        return element;
+
+        APP.store.dispatch(
+            showNotification(ParticipantNotification, {
+                title: displayName,
+                defaultTitleKey: displayNameKey,
+                descriptionKey: messageKey,
+                descriptionArgs: messageArguments
```

Comment:
This isn't in react package, but we can sort alphabetically, so that it's ready to be moved to react whenever we're ready with other things in MessageHandler.